javacutil/src/main/java/org/checkerframework/javacutil/AnnotationMirrorMap.java

```diff
@@ -175,4 +181,39 @@ public int hashCode() {
     for (Entry<AnnotationMirror, V> entry : entrySet()) result += entry.hashCode();
     return result;
   }
+
+  /**
```

Comment:
I like to put all the constructors and factory methods together, at the top of the file.